tests/test_compute.py

```diff
@@ -297,6 +297,91 @@ def test_compute(client, publisher_wallet, consumer_wallet, free_c2d_env):
     assert result_data is not None, "We should have a result"
 
 
+@pytest.mark.integration
+def test_compute_arweave(client, publisher_wallet, consumer_wallet, free_c2d_env):
+    valid_until = get_future_valid_until()
+    ddo, tx_id, alg_ddo, alg_tx_id = build_and_send_ddo_with_compute_service(
+        client,
+        publisher_wallet,
+        consumer_wallet,
+        False,
```

Comment:
please add the arg name key here for clarity (I assume it's diff_provider or something)

Reply:
Done.